tests/integration/test_logcollector/test_configuration/test_basic_configuration_location.py

```diff
@@ -0,0 +1,176 @@
+
+test_win_configuration, test_win_metadata, test_win_cases_ids = configuration.get_test_cases_data(win_cases_path)
+test_win_configuration = configuration.load_configuration_template(default_config_path, test_win_configuration, test_win_metadata)
+
+
+if sys.platform == MACOS:
+    test_configuration += test_macos_dup_configuration
+    test_metadata += test_macos_metadata
+    test_cases_ids += test_macos_cases_ids
+if sys.platform == WINDOWS:
+    test_configuration += test_win_configuration
+    test_metadata += test_win_metadata
+    test_cases_ids += test_win_cases_ids
+
+# Test daemons to restart.
+daemons_handler_configuration = {'all_daemons': True}
+
+# Test function.
+@pytest.mark.parametrize('test_configuration, test_metadata', zip(test_configuration, test_metadata), ids=test_cases_ids)
+def test_configuration_location(test_configuration, test_metadata, truncate_monitored_files,
+                                set_wazuh_configuration, daemons_handler):
+    '''
+    description: Check if the 'wazuh-logcollector' daemon starts properly when the 'location' tag is used.
+                 For this purpose, the test will configure the logcollector to monitor a 'syslog' directory
+                 and use a pathname with special characteristics. Finally, the test will verify that the
+                 Wazuh component is started by checking its process, and the Wazuh API returns the same
+                 values for the 'localfile' section that the configured one.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_logcollector:
+            type: fixture
+            brief: Clear the 'ossec.log' file and start a new monitor.
+
+    assertions:
+        - Verify that the Wazuh component (agent or manager) can start when the 'location' tag is used.
+        - Verify that the Wazuh API returns the same value for the 'localfile' section as the configured one.
+        - Verify that the expected event is present in the log file.
+
+    input_description: A configuration template (test_basic_configuration_location) is contained in an external
+                       YAML file (wazuh_basic_configuration.yaml). That template is combined with different
+                       test cases defined in the module. Those include configuration settings for
+                       the 'wazuh-logcollector' daemon.
+
+    expected_output:
+        - Boolean values to indicate the state of the Wazuh component.
+        - Did not receive the expected "ERROR: Could not EvtSubscribe() for ... which returned ..." event.
+
+    tags:
+        - invalid_settings
+    '''
+
+    if sys.platform == WINDOWS:
+        assert check_if_process_is_running(WAZUH_AGENT_WIN) == True
```

Comment:
Maybe you can do this with a fixture looking for the expected starting log. Check for example the function `wait_for_analysisd_startup`. If you create this fixture, place it in `tests/integration/test_logcollector/conftest.py` as it may be useful to other logcollector suites.